plugin/assets/src/wizard/components/notice/index.js

```diff
@@ -22,7 +22,7 @@
 const Notice = props => {
 	return (
 		<div className={ `notice material-wizard__notice ${ props.type }` }>
-			<p>{ props.message }</p>
+			<p dangerouslySetInnerHTML={ { __html: props.message } }></p>
```

Comment:
Why does this need to be dangerous set?

Reply:
Hey @rodydavis, We are sending an error with the anchor tag to set the API key from the material setting page when the user clicks update font button:

We can: 
1. Remove the link and keep it a text error message (and also remove `dangerouslySetInnerHTML`).
1. Use the library to sanitize HTML such that it only allows specified tags. https://github.com/cure53/DOMPurify 

We are sanitizing it from PHP before passing it to the frontend here: https://github.com/material-components/material-design-for-wordpress/pull/165/files#diff-fd8d3142c003529c626daa8443ca0b86e5d6e29bf3d9a8cf4d3053d2cf65ba5aR225-R232

Reply:
Yeah lets do option 2